temp.js

```diff
@@ -0,0 +1,138 @@
+const {
+    Worker, isMainThread, workerData, BroadcastChannel, MessageChannel,
+  } = require('worker_threads');
+  const CoinbasePro = require('coinbase-pro');
+  const stringTable = require('string-table');
+  const moment = require('moment');
+  const log4js = require('log4js');
+  const { v4 } = require('uuid');
+  const { wsUrl } = require('./model/constants');
+  const { strategies } = require('./strategies/all_strategies');
+  const { loadConfigurationFile } = require('./utils/loadConfigurationFile');
+  const { checkEnvironmentVariables } = require('./utils/checkEnvironmentVariables');
+  
+  try {
+    
+    if (isMainThread) {
+      checkEnvironmentVariables(process.env);
+      const appConfiguration = loadConfigurationFile(process.argv);
+      const client = new CoinbasePro.AuthenticatedClient(
+        process.env.apiKey,
+        process.env.apiSecret,
+        process.env.apiPassphrase,
+      );
+      const { port1, port2 } = new MessageChannel();
+      const tickerChannel = new BroadcastChannel('ticker');
+      const candleChannel = new BroadcastChannel('candles-every-minute-past-10-minutes');
+      const logAppenders = {};
+      const appendersName = ['main'];
+      const logCategories = {};
+      // logging configuration
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 6.](https://app.codacy.com/gh/amcalabretta/botbase/pullRequest?prid=7628479)
```suggestion
    // logging configuration
```